gemfiles/activesupport_5.gemfile.lock

```diff
@@ -0,0 +1,58 @@
+PATH
```

Comment:
@invocandg Additionally when you're updating the github workflows, we typically don't commit the lock files for appraisals so that the test will always run with the latest minor/patches for the appraised gems. Would you mind adding `gemfiles/*.lock` to the `.gitignore` and removing these from git tracking?